packages/playwright-core/src/utils/registry.ts

```diff
@@ -126,6 +104,8 @@ const DOWNLOAD_PATHS = {
     'mac10.15': 'builds/firefox-beta/%s/firefox-beta-mac-11.zip',
     'mac11': 'builds/firefox-beta/%s/firefox-beta-mac-11.zip',
     'mac11-arm64': 'builds/firefox-beta/%s/firefox-beta-mac-11-arm64.zip',
+    'mac12': 'builds/firefox-beta/%s/firefox-beta-mac-11.zip',
```

Comment:
Should be -mac12.zip right?

Reply:
for everything but webkit we re-use macos 11 builds